### ISSUE TITLE
Bug with new syntax highlighting - AND is not highlighted

### ISSUE THREAD

**Issue by AlbusPortucalis** [jcaillon/3P#295]
Thanks for updating 3P so fast. It facilitates programming in Progress a lot. I share below a bug I saw with the new update.

![image](https://user-images.githubusercontent.com/54470604/176452102-36596e72-742c-45c1-bd1f-d555b2d3143e.png)

**Comment by jcaillon** [comment]
Hi @AlbusPortucalis , glad that it is useful for you. 

Took a few minutes to look at the bug. Actually you accidently found an issue that has been there for the very beggining x) Let me explain...

The screencap you attached is actually totally legit; this is how this theme is programmed; `Operator	#000000	@Default	0` operator keywords like AND OR... are in black on default background and with no font weight.

The issue is that if you try the different syntax highlighting theme through the UI:
![image](https://user-images.githubusercontent.com/11553075/176475639-0a2325a5-24b9-4763-84bb-47709d7ba2dd.png)

then you will see soemthing like that for dev studio theme:

![image](https://user-images.githubusercontent.com/11553075/176475739-449ebb6f-0696-4548-9322-3e4f6e1ae4b8.png)

and if you restart notepad++ after that, you will see it like that:

![image](https://user-images.githubusercontent.com/11553075/176475830-d9510594-50eb-4c83-a9dd-deda12cb4e59.png)

The difference is that some of the highlighted words are no longer in **BOLD**.

The actual bug is that, when switching the themes, i do not correctly reset the font size, giving the impression that some words are bold for the dev studio theme where everything is actually in normal font.

---

TL;DR; I will correct the bug and modify the theme so that the operators are more visible.

In the mean time, you can [use this page ](https://jcaillon.github.io/3P/#/custom-themes) to learn how to customize the style in question. Here is my proposal (i just changed the last digit, 2 for bold and 4 for italic):

```
> Developer studio
Default	#000000	#FFF	0
Comment	#3F7F5F	@Default	0
SingleLineComment	@Comment	@Default	0
Preprocessor	#B3A68E	@Default	4
JumpStatement	#7F0055	@Default	2
Statement	#7F0055	@Default	2
Type	#CD3A3A	@Default	2
Keyword	#7F0055	@Default	2
Operator	#000000	@Default	2
Abbreviation	#EC4D00	@Default	2
Include	#FF0080	@Default	4
DoubleQuote	#2A00FF	@Default	0
SimpleQuote	#2A00FF	@Default	0
NormedVariables	#FF8000	@Default	0
Number	#2A00FF	@Default	0
CaretLine	#000000	#E8E8FF	0
Selection	#000000	lightgrey	0
WhiteSpace	#CCC	@Default	0
NoError	#007B74	#C6EFCE	0
Error0	#3F3F3F	#F2F2F2	0
Error1	#9C6500	#FFEB9C	0
Error2	#833C0C	#FFCC99	0
Error3	#9C0006	#FFC7CE	0
Error4	#58267E	#CC99FF	0
CaretColor	@Default	#000000	0
FoldMargin	#595959	#F3F3F3	0
FoldActiveMarker	@SimpleQuote	#000000	0
LineNumberMargin	#595959	#E4E4E4	0
SmartHighLighting	#09FF00	#000000	0
BraceHighLight	#333	#5EFF00	0
BadBraceHighLight	#FFFFFF	#FF42FF	0
```

**Comment by AlbusPortucalis** [comment]
Got it! Thanks for the detailed answer.
